toolkit/scripts/containerized-build/create_container_build.sh

```diff
@@ -23,38 +23,36 @@ print_error() {
 help() {
 echo "
 Usage:
-sudo make containerized-rpmbuild [REPO_PATH=/path/to/CBL-Mariner] [MODE=test|build] [VERSION=1.0|2.0] [MOUNTS= /src/path:/dst/path] [ENABLE_REPO=y]
+sudo make containerized-rpmbuild [REPO_PATH=/path/to/CBL-Mariner] [MODE=test|build] [VERSION=1.0|2.0] [MOUNTS=/path/in/host:/path/in/container] [BUILD_MOUNT=/path/to/build/chroot/mount] [ENABLE_REPO=y]
 
 Starts a docker container with the specified version of mariner.
 
 Optional arguments:
-    REPO_PATH:      path to the CBL-Mariner repo root directory. default: "current directory"
-    MODE            build or test. default:"build"
-                        In 'test' mode it will use a pre-built mariner chroot image.
-                        In 'build' mode it will use the latest published container.
-    VERISION        1.0 or 2.0. default: "2.0"
-    MOUNTS          mount a directory into the container. Should be of the form '/src/dir:/dest/dir'. For multiple mounts, please use ',' as delimiter
-                        e.g. MOUNTS=/src/dir1:/dest/dir1,/src/dir2:/dest/dir2
-    ENABLE_REPO:    Set to 'y' to use local RPMs to satisfy package dependencies. default: "n"
+    REPO_PATH:      path to the CBL-Mariner repo root directory. default: current directory
+    MODE            build or test. default: build
+                        In 'build' mode it will use a pre-built Mariner chroot image.
+                        In 'test' mode it will use the latest Mariner published container.
+    VERISION        1.0 or 2.0. default: 2.0
+    MOUNTS          mount a host directory into the container. Should be of the form '/host/dir:/container/dir'. For multiple mounts, please use \" \" as delimiter
+                        e.g. MOUNTS=\"/host/dir1:/container/dir1 /host/dir2:/container/dir2\"
+    BUILD_MOUNT     path to mountpoint for container's BUILD and BUILDROOT directories. default \$REPO_PATH/build
```

Comment:
I'm a bit leery of mounting all of `./build` into the container, there are a lot of sensitive bits in there (chroots etc.), and we might also have some interesting interactions with overlays etc. (you can't overlay on another overlay). I think this should probably be its own subfolder.

Reply:
Sorry, this echo line is misleading. The BUILD and BUILDROOT are mounted at build/container-build and build/container-buildroot. If the user provides a BUILD_MOUNT, the script will create ${BUILD_MOUNT}/container-build and ${BUILD_MOUNT}/container-buildroot folders, and use those as mountpoints.
So, I guess this line should read
BUILD_MOUNT     path to folder to **create** mountpoints for container's BUILD and BUILDROOT directories. **mountpoints will be ${BUILD_MOUNT}/container-build and ${BUILD_MOUNT}/container-buildroot.** default \$REPO_PATH/build.
Does that convey the message?